data/resources/style.css

```diff
@@ -213,7 +213,7 @@ messagesticker messageindicators,
 .osd-indicator {
   background-color: alpha(black, 0.4);
   color: white;
-  opacity: 1.0; /* Reset opacity from message bubble */
+  opacity: 1; /* Reset opacity from message bubble */
```

Comment:
Can you put this separate changes in a separate commit?

Reply:
Ok, that's just formatted by my formatter)